src/OrchardCore/OrchardCore.Logging.NLog/WebHostBuilderExtensions.cs

```diff
@@ -19,6 +19,8 @@ public static IWebHostBuilder UseNLogWeb(this IWebHostBuilder builder)
             .ConfigureAppConfiguration((context, _) =>
             {
                 var environment = context.HostingEnvironment;
+                var appData = System.Environment.GetEnvironmentVariable("ORCHARD_APP_DATA");
+                var configDir = string.IsNullOrWhiteSpace(appData) ? $"{environment.ContentRootPath}/App_Data" : appData;
```

Comment:
same as above

Reply:
same as above can't access the private constant